src/PreCICE.jl

```diff
@@ -1332,5 +1340,149 @@ function actionReadIterationCheckpoint()
     return unsafe_string(msgCstring)
 end
 
+@doc """
+
+        isGradientDataRequired(dataID::Integer)
+        
+Checks if the given data set requires gradient data. We check if the data object has been intialized with the gradient flag.
+# Arguments
+- `dataID::Integer`: ID of the data to be checked. Obtained by [`getDataID`](@ref).
+
+"""
+function isGradientDataRequired(dataID::Integer)
+    return ccall((:precicec_isGradientDataRequired, "libprecice"), Cbool, dataID)
+end
+
+@doc """
+
+writeBlockVectorGradientData(dataID::Integer, valueIndices::AbstractArray{Cint}, gradientValues::AbstractArray{Float64})
```

Comment:
This line needs to be indented for Julia to know that it is the function call and mark it as such